solana/pyth2wormhole/client/src/main.rs

```diff
@@ -190,301 +225,276 @@ fn handle_attest(
         .collect();
     let batch_count = batches.len();
 
-    // NOTE(2022-04-26): only increment this if `daemon` is false
-    let mut finished_count = 0;
+    let rpc_cfg = Arc::new(RLMutex::new(
+        RpcCfg {
+            url: rpc_url,
+            timeout: confirmation_timeout,
+            commitment: commitment.clone(),
+        },
+        rpc_interval,
+    ));
```

Comment:
From a readibility perspective, the fact that an `RLMutex` is used here may be missed at first glance by someone skimming over the code. It may be more obvious to make the `RLMutex` a field and highlight this with a comment.

Reply:
I think that's a good idea, thanks